test/unit/translators/twitter_translators/tweet_translator_test.rb

```diff
@@ -1,12 +1,12 @@
-require "test_helper"
+require 'test_helper'
 
 class TwitterTranslatorTest < ActiveSupport::TestCase
   def build_twitter_user
     OpenStruct.new(
-      id: "my_twitter_user_id",
-      name: "my_twitter_user_name",
-      email: "my_twitter_user_email@example.org",
-      screen_name: "my_twitter_screen_name",
+      id: 'my_twitter_user_id',
+      name: 'my_twitter_user_name',
+      email: 'my_twitter_user_email@example.org',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.